python/python/glide/async_commands/redis_modules/search/index.py

```diff
@@ -0,0 +1,80 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+from enum import Enum
+from typing import List, Optional
+
+
+class IndexType(Enum):
+    HASH = "HASH"
+    JSON = "JSON"
+
+
+class Index:
+    """
+    Class for defining an index in a schema.
+
+    Args:
+        index (str): The name of the index.
+        index_type (IndexType): The type of the index, supports HASH (default) and JSON. To index JSON,
```

Comment:
don't document what options IndexType has, as it might changed and then we will need to maintain it here too.